code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -301,6 +305,66 @@
 		qdel(S)
 	return ..()
 
+/obj/item/nullrod/scythe/talking/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/soulstone) && possessed)
+		if(!(obj_integrity < max_integrity))
+			to_chat(user, "<span class='notice'>You have no reason to replace a perfectly good soulstone with a new one.</span>")
+			return
+		to_chat(user, "<span class='notice'>You load a new soulstone into the possessed blade.</span>")
+		playsound(user, 'sound/weapons/gun_interactions/shotgunpump.ogg', 60, 1)
+		obj_integrity = max_integrity
+		for(var/mob/living/simple_animal/shade/sword/sword_shade in contents)
+			sword_shade.health = sword_shade.maxHealth
+		qdel(I)
+		return
+	return ..()
+
+/obj/item/nullrod/scythe/talking/take_damage(damage_amount)
+	if(possessed)
+		for(var/mob/living/simple_animal/shade/sword/sword_shade in contents)
+			sword_shade.take_overall_damage(damage_amount)
+	return ..()
+
+/obj/item/nullrod/scythe/talking/proc/click_actions(atom/attacking_atom, mob/living/simple_animal/attacking_shade)
+	if(world.time <= attacking_shade.next_move) // yea we gotta check
+		return
+	if(!istype(attacking_atom, /atom/movable))
+		return
+	attacking_shade.changeNext_move(CLICK_CD_MELEE)
+	if(!isturf(loc))
+		if(ismob(loc))
+			var/mob/living/carbon/human/our_location = loc
+			if(istype(our_location))
+				if(!(src == our_location.l_hand) && !(src == our_location.r_hand))
+					return
+				if(our_location.Adjacent(attacking_atom)) // with a buddy we deal 10 damage :D
+					our_location.do_attack_animation(attacking_atom, used_item = src)
+					melee_attack_chain(attacking_shade, attacking_atom)
+				return
+	if(Adjacent(attacking_atom)) // without a buddy we only deal 5 damage :c
+		force = force - 5
+		do_attack_animation(attacking_atom, used_item = src)
+		melee_attack_chain(attacking_shade, attacking_atom)
+		force = force + 5
+
+/mob/living/simple_animal/shade/sword/create_mob_hud()
+	hud_used = new /datum/hud/sword(src)
+
+/datum/hud/sword/New(mob/user)
+	..()
+
+	mymob.healths = new /obj/screen/healths()
+	infodisplay += mymob.healths
+
+/mob/living/simple_animal/shade/sword/ClickOn(atom/A, params)
+	if(client.click_intercept)
```

Comment:
```suggestion
	if(client?.click_intercept)
```

Reply:
if they don't have a client but clicked on something we want to know that's the case

Reply:
Fair